### PR TITLE
Add OVM v0.6 contracts and tests

### PR DESCRIPTION
**Pull request by krebernisak**

### CODE REVIEW

File: evm-contracts/src/v0.6-ovm/v0.6-ovm.diff [hunk 1]
```diff
@@ -0,0 +1,1500 @@
+diff --git a/src/v0.6/AccessControlledAggregator.sol b/src/v0.6-ovm/AccessControlledAggregator.sol
```
Comment (HenryNguyen5) [inline evm-contracts/src/v0.6-ovm/v0.6-ovm.diff:1]: Is this file needed?

**Review by HenryNguyen5 — CHANGES_REQUESTED** [review]

File: evm-contracts/src/v0.6-ovm/v0.6-ovm.diff (hunk 1 above)
Comment (krebernisak) [reply evm-contracts/src/v0.6-ovm/v0.6-ovm.diff:1]: Probably not in the long term. For now, I've included it as it might help assess the changes between our `v6` contracts and the new `v6-ovm` for audit purposes.

**Comment by krebernisak** [comment]
Closing (for now) in favor of [Add OVM v0.6 contracts and tests [Node v10] #3719](https://github.com/smartcontractkit/chainlink/pull/3719/files).
